data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageProcessor.java

```diff
@@ -14,20 +14,42 @@
 
 package com.wipro.ats.bdre.lineage;
 
+import com.wipro.ats.bdre.md.dao.LineageQueryDAO;
+import com.wipro.ats.bdre.md.dao.jpa.LineageQuery;
+import com.wipro.ats.bdre.md.dao.jpa.LineageQueryType;
 import org.apache.hadoop.hive.ql.parse.ParseException;
 import org.apache.hadoop.hive.ql.parse.SemanticException;
 import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.AutowireCapableBeanFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 import java.io.IOException;
+import java.util.Date;
+import java.util.UUID;
 
 /**
  * Created by jayabroto on 21-05-2015.
  */
 public class LineageProcessor {
 	private static final Logger LOGGER = Logger.getLogger(LineageProcessor.class);
+	private String id;
 
 	private String defaultHiveDbName = LineageConstants.defaultHiveDbName;              // default Hive db
 
+	public LineageProcessor() {
+		//this.defaultHiveDbName = defaultHiveDbName;
+		 /*Hibernate Auto-wire*/
+		ApplicationContext context = new ClassPathXmlApplicationContext("spring-dao.xml");
+		AutowireCapableBeanFactory acbFactory = context.getAutowireCapableBeanFactory();
+		acbFactory.autowireBean(this);
+		LOGGER.debug("--------------Auto wiring initiated---------------");
+	}
+
+	@Autowired
+	LineageQueryDAO lineageQueryDAO;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)